src/ray/raylet/node_manager.cc

```diff
@@ -1318,6 +1319,9 @@ void NodeManager::TreatTaskAsFailedIfLost(const Task &task) {
 
 void NodeManager::SubmitTask(const Task &task, const Lineage &uncommitted_lineage,
                              bool forwarded) {
+  // TODO(qwang): How to get host address.
```

Comment:
Is this TODO still needed?

Reply:
I think we still need this TODO. It can be removed after we address rayConfig type issue.